Improve “User Center” documentation (#160)
SEKOIA-IO/documentation: docs/user_center/notifications.md

# User Center: Notification Rules

> SEKOIA.IO’s User Center will let you configure “Notification Rules” so you can keep informed of new activity that happens on SEKOIA.IO. Notifications Rules will let you focus on specific SEKOIA.IO events (Operations Center alerts and Intelligence Center reports), add precise conditions and configure actions to be triggered (send an e-mail, display a notification in SEKOIA.IO, etc.).

Reviewer: ```suggestion
SEKOIA.IO’s User Center let you configure “Notification Rules” so you can be kept informed of new activities that happens on SEKOIA.IO. Notifications Rules will let you focus on specific SEKOIA.IO events (Operations Center alerts and Intelligence Center reports), add precise conditions and configure actions to be triggered (send an e-mail, display a notification in SEKOIA.IO, etc.).
```

![Create a New Notification Rule](../assets/user_center/notification_rules.png)

This documentation will let you go deeper into SEKOIA.IO’s notification mechanism.

## Available Conditions

To write a new notification rule, the first step consists in selecting an “event” and conditions that will trigger your notification rule.

### Operations Center Alerts

> You can configure your notification rule to be triggered on alerts (that can be either new or already existing). Then, you’ll have to choose one or multiple conditions amongst multiple options (alert status, its urgency, asset concerned by the alert, etc.).

Reviewer 2: ```suggestion
You can configure your notification rule to be triggered on alerts (that can be either new or already existing). Then, you’ will be able to add one or multiple conditions amongst multiple options (alert status, its urgency, asset concerned by the alert, etc.).
```

For example, you can decide to trigger a notification when the following criteria are met:

- Status of the new alert is “Ongoing”.
- Urgency of the new alert is higher than 80.
- Involved asset is “My Critical Asset”.

### Intelligence Center Reports

FIXME

## Triggered Actions

> When your conditions are configured, you want to decide which action should be triggered by then notification mechanism. You can enable one or more action for each notification rule.

Reviewer 2: ```suggestion
When your conditions are configured, you want to decide which action should be triggered by the notification mechanism. You can enable one or more action for each notification rule.
```

For example you can decide to send an e-mail and to send a message on a Mattermost channel.

### SEKOIA.IO Notification

> The “Notification” action allows you to send an “in-app” notification that will be made available across SEKOIA.IO. All notifications can be accessed from the “bell” icon on the top right. By clicking there, the notification panel with all notifications will be displayed.

Reviewer 2: ```suggestion
The “Notification” action allows you to send an “in-app” notification that will be available across SEKOIA.IO. All notifications can be accessed from the “bell” icon on the top right. By clicking there, the notification panel with all notifications will be displayed.
```
